packages/backend/server/src/core/doc/adapters/workspace.ts

```diff
@@ -600,54 +433,16 @@ export class PgWorkspaceDocStorageAdapter extends DocStorageAdapter {
    * keep in next release to avoid downtime when upgrading instances
    */
   private async getUpdateSeq(workspaceId: string, guid: string, batch = 1) {
-    const MAX_SEQ_NUM = 0x3fffffff; // u31
```

Comment:
directly remove seq usage

Reply:
The `seq` has been deleted, there are quite a few changes, need to take another look.